MC/bin/o2_hybrid_gen.py

```diff
@@ -69,6 +69,19 @@ def main():
     else:
         print(f"Running in {mode} mode")
 
+    # Available options for trigger are "off", "or", "and"
+    # in all the other cases the trigger is forced "off"
+
+    trgbase = {
```

Comment:
we could print this only when the user asks for it. Otherwise it might bloat the JSON quickly.

Reply:
I will add a --trigger flag.